modules/460-log-shipper/docs/USAGE.md

```diff
@@ -176,3 +176,25 @@ spec:
   type: loki
   url: http://loki.loki:3100
 ```
+
+
+# Elasticsearch <= 6.X usage
```

Comment:
It would help if you used the double `#` sign here, and the same goes for the Russian doc.
```suggestion
## Elasticsearch <= 6.X usage
```

![image](https://user-images.githubusercontent.com/32434187/146029909-39f9ab3f-336f-4a08-9427-1fb2a7e7521e.png)